Dockerfile

```diff
@@ -1,9 +1,3 @@
-ARG REACT_APP_AUTH_REDIRECT_URL
```

Comment:
We sure this isn't gonna break things for when we customise it for dev environment usage?

Reply:
Resolved via slack.
This was a duplicate, we still have it